### ISSUE TITLE
Publishing status needs to be set to "In production" to avoid 403

### ISSUE THREAD

**Issue by filipmrc** [kidd/org-gcal.el#140]
It seems as if the publishing status needs to be set to "In production" when setting up the OAuth consent screen. Without this `org-gcal-fetch` results in a 403 for me.

This only happened on one of my google accounts, so only some people may get this error.

![screencap](https://user-images.githubusercontent.com/12412582/111062800-80328280-84ab-11eb-9981-d59e5b3e44c6.png)

**Comment by telotortium** [comment]
Hi, where would the best place to add this step be in the [Installation section of the README](https://github.com/kidd/org-gcal.el/blob/d38acdad70883dc3fe34a4cd41024e5a8099ddb2/README.org#Installation)?

**Comment by Artraxon** [comment]
I'd advise against that because google is probably going to try to verify your app.
It is way easier to keep it in testing and add yourself as a test-user, in this case you also gain access.

**Comment by arifer612** [comment]
I agree with what @Artraxon has to say about keeping the project in testing mode
and allowing only yourself to be the test-user. However, there may be use-cases
where you may want to share the calendar with others, so both cases should be
included in the documentation. I had just gone through with the installation
process so I can still remember what I had to do to get myself past the 403
error. I think adding this as the 6th step would be ideal, since it can be done
right after setting up the content screen.

Returning back to the APIs & Services tab, select OAuth Consent Screen and under
a "Test User" header, click on the dial to add new test users underneath it.
Input your email address and you can then proceed with the rest of the instructions.

**Comment by fcevado** [comment]
besides what @Artraxon mentioned, currently the `google calendar` api is not enabled by default. so you need to enabled it, otherwise you gonna get a 403.
